vsphere/cpi.yml

```diff
@@ -48,14 +48,22 @@
     password: ((vcenter_password))
     datacenters:
     - name: ((vcenter_dc))
-      vm_folder: ((vcenter_vms))
-      template_folder: ((vcenter_templates))
-      datastore_pattern: ((vcenter_ds))
-      persistent_datastore_pattern: ((vcenter_ds))
-      disk_path: ((vcenter_disks))
-      clusters:
-      - ((vcenter_cluster)): {}
+      vm_folder: ((vcenter_vm_folder))
```

Comment:
I found them a bit misleading. It also lacks the ability to have the ephemeral disks on a different datastore than the persistent disks. In my opinion the changed names are more expressive.